plugins/draw-tools.js

```diff
@@ -502,7 +515,7 @@ window.plugin.drawTools.optImport = function() {
 window.plugin.drawTools.optReset = function() {
   var promptAction = confirm('All drawn items will be deleted. Are you sure?', '');
   if(promptAction) {
-    delete localStorage[window.plugin.drawTools.KEY_STORAGE];
+    localStorage[window.plugin.drawTools.KEY_STORAGE] = '[]';
```

Comment:
Why?

Reply:
Just like in upstream [draw-tools](https://github.com/IITC-CE/ingress-intel-total-conversion/blob/master/plugins/draw-tools.js#L493).

This PR only replaces [`679939c` (#406)](https://github.com/IITC-CE/ingress-intel-total-conversion/pull/406/commits/679939c461b61b2fc9701c7279de8a2df9d4c232) the key with a constant

Reply:
Ok, that let's continue in origin: https://github.com/IITC-CE/ingress-intel-total-conversion/commit/940ff669aaf08e1bdac2b5c4cbcdd9e06f3cc1d1#r45236089

Reply:
The task is NOT to delete the KEY_STORAGE, but to provide an empty JSON-String (nothing drawn yet).
The value of KEY_STORAGE is changing depending on the selected MPE-Project. 
To be able select the active dataset transparently fro the plugin, i.e. without the orginal plugin to know, which storage is currently in use. Only MPE is aware of the project names, not the plugin.